### PR TITLE
Base code added

### PR DESCRIPTION
**Pull request by ANSHUMAN87**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
```
Comment (sharathks84) [inline build.gradle:1]: No Reade Me File
No license File
No Copyright information

**Review by sharathks84 — COMMENTED** [review]
No Reade Me File
No license File
No Copyright information

**Review by sharathks84 — APPROVED** [review]
OK, approved

**Review by sharathks84 — APPROVED** [review]
OK approved

**Review by Gleelakrishna — APPROVED** [review]
ok

**Pull request merged**